nucypher/network/nodes.py

```diff
@@ -448,25 +422,12 @@ def remember_node(self,
                 # This node is already known.  We can safely return.
                 return False
 
-        self.known_nodes.record_node(node)  # FIXME - dont always remember nodes, bucket them.
-
-        if self.save_metadata:
-            self.node_storage.store_node_metadata(node=node)
+        self.known_nodes.record_node(
+            node
+        )  # FIXME - dont always remember nodes, bucket them.
```

Comment:
is there an issue for this?

Reply:
Yes, it's a very old issue https://github.com/nucypher/nucypher/issues/567

Although it's somewhat resolved by PR #3382